### PR TITLE
Use pyproj instead

### PR DESCRIPTION
**Pull request by Charlie-Henry**
Was still having problems building the docker image with geopandas so I've used pyproj instead to do the transforming of coordinates.

### CODE REVIEW

**Comment by Charlie-Henry** [comment]
third times the charm @johnclary @chiaberry 👀 ?

**Review by johnclary — APPROVED** [review]
this looks very awesome! thank you for finding yet another way to do this/ TIL. good stuff!

**Comment by johnclary** [comment]
cc @frankhereford bc i know he likes to be aware of GIS tooling :)

File: csr/csr_to_socrata.py [hunk 1]
```diff
@@ -63,23 +63,14 @@ def extract():
...
-        ),
-        crs="ESRI:102739",
-    )
-    gdf = gdf.to_crs("EPSG:4326")
-
-    # Get wgs84 location columns
-    df["latitude"] = gdf["geometry"].y
-    df["longitude"] = gdf["geometry"].x
+    # projection of coordinates
+    transformer = Transformer.from_crs(crs_from="ESRI:102739", crs_to="EPSG:4326")
```
Comment (frankhereford) [inline csr/csr_to_socrata.py:71]: Thanks for the @ @johnclary! 

I couldn't recall having seen SRID 102739 anytime recently. I looked it up, and it's a deprecated version of our Central State Plan projection that works great for us here in ATX and is used a lot in the department. I pulled up the proj4 string for it and compared it to 2277, and they are the same for all intents and purposes - so it should just work for you. If we have that data somewhere under our control that you're transforming from, we might want to update its metadata by reprojecting it into 2277. Not a big deal, but interesting at least. I'd be a house-keeping thing.

**Pull request merged**